### PR TITLE
[9.0][FIX] report_qweb_pdf_watermark

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Backport of https://github.com/OCA/reporting-engine/pull/144.

### CODE REVIEW

File: report_qweb_pdf_watermark/models/report.py [hunk 1]
```diff
@@ -3,11 +3,20 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from base64 import b64decode
 from logging import getLogger
-from pyPdf import PdfFileWriter, PdfFileReader
+from pyPdf.pdf import PdfFileWriter, PdfFileReader
```
Comment (astirpe) [inline report_qweb_pdf_watermark/models/report.py:6]: Why this line was changed?

**Review by astirpe — CHANGES_REQUESTED** [review]

File: report_qweb_pdf_watermark/models/report.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply report_qweb_pdf_watermark/models/report.py:6]: Because otherwise my PyCharm says 'unresolved reference...'  😙

**Comment by AaronHForgeFlow** [comment]
image
This error is showing in all the Pull Request in v9. This PR is meant to solve that but it's not working. Can anybody the label Needed help here?

**Comment by astirpe** [comment]
Can you try to backport the entire commit https://github.com/OCA/reporting-engine/commit/7d621022b00b473b734f1e0d7cb19727012be8af? I mean deleting also the obsolete rows in `.travis.yml`.

I'm not sure this will fix the problem though.

**Comment by MiquelRForgeFlow** [comment]
I fully backported the two commits of that PR but both squashed into one.

**Comment by astirpe** [comment]
@mreficient it seems you missed a commit. Can you double check? Take a look at the commit mentioned in my previous post and then look at your commits.

**Comment by MiquelRForgeFlow** [comment]
https://github.com/OCA/reporting-engine/pull/154/commits/623f86e2672d0132bde0ab1a2fe2529ca53804bd is a squashed version of https://github.com/OCA/reporting-engine/pull/144/commits/6dacecae08baf881a3e0c7f9cd665e464fe41ddf and https://github.com/OCA/reporting-engine/pull/144/commits/7d621022b00b473b734f1e0d7cb19727012be8af.

**Comment by hbrunn** [comment]
this finally is fixed in #141 by now

**Pull request closed without merging**